openedx/features/ucsd_support/tests/tests.py

```diff
@@ -0,0 +1,55 @@
+import json
+import mock
+
+from django.test import TestCase
+from django.urls import reverse
+from django.test import Client
+
+
+class TestEmailSendFromSupport(TestCase):
+    """
+    Tests email sending via contact/support form.
+    """
+
+    @classmethod
+    def setUpClass(self):
+        self.client = Client()
+        self.send_mail_url = reverse('ucsd_support_email')
+        self.success_content = 201
+        self.fail_content = 0
+
+    def test_send_email_successful(self):
+        """
+        Make sure email works successfully.
+        """
+
+        test_email = {'subject': 'Subject goes here',
```

Comment:
Please remove duplication of `test_email`

Reply:
done